extension/src/openvic-extension/classes/GFXButtonStateTexture.cpp

```diff
@@ -181,8 +172,7 @@ void GFXButtonStateHavingTexture::_clear_button_states() {
 
 GFXButtonStateHavingTexture::GFXButtonStateHavingTexture() : button_image {}, button_state_textures {} {}
 
-Ref<GFXButtonStateTexture> GFXButtonStateHavingTexture::get_button_state_texture(
-	GFXButtonStateTexture::ButtonState button_state
+Ref<GFXButtonStateTexture> GFXButtonStateHavingTexture::get_button_state_texture(GFXButtonStateTexture::ButtonState button_state
 ) {
```

Comment:
This is breaking just the `) {` onto a new line to avoid going over 128 columns wide, but IMO it looks far better either with no line break or with breaks either side of the `()` around the arguments, as it is now I find it difficult to read at a glance.